tests/stack/test_api.py

```diff
@@ -529,3 +578,84 @@ def test_download_resource(self, stack_api):
             api.RESOURCE_PROPERTIES: {'test': 'test'}
         }
         stack_api._download_resource(resource_badservice)
+
+    def test_deploy_valid_stack_status(self, stack_api, tmpdir):
+        """
+            Test valid creation of the stack status on a correctly setup stack
```

Comment:
Improve the descriptions?
"Stack status should be valid when ...."
from the current description is is unclear what to expect